src/API/Program.cs

```diff
@@ -0,0 +1,95 @@
+using Core.Interfaces;
+using Infrastructure.Repository;
+using Infrastructure.Wrapper;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using System;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.WebHost.ConfigureKestrel(options =>
+{
+    options.ListenAnyIP(5000);
+});
+
+builder.Services.AddControllers();
+
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy("AllowAllOrigins", builder =>
+    {
+        builder.AllowAnyOrigin()
```

Comment:
Na dev akceptowalne, produkcyjnie zależy od scenariusza - jak to API pod konkretną aplikację, rozwżyłbym ustawienie, jeżeli publiczne to jest ok.